docs/team/shared-infrastructure.md

```diff
@@ -0,0 +1,27 @@
+# Our Shared Infrastructure
+
+There are a few services and infrastructure that the JupyterHub team has access to.
+Some of these are useful for our development workflows, while others are used as a part of services that we run.
+
+## GitHub
+
+We have [a GitHub organization](https://github.com/jupyterhub/) for hosting all of our code repositories.
+This organization is where we do most of the code-related work for the project, and where we have discussions and coordination.
+
+## Image registry on quay.io
+
+We have [a quay.io organization](https://quay.io/organization/jupyterhub) for hosting Docker images.
+This is used to push new images for tools like BinderHub and JupyterHub for Kubernetes.
```

Comment:
It's a bit more complicated at the moment. Current situation:
- repo2docker: pushed to quay.io only due to problems with accessing the jupyter Docker Hub account
- BinderHub and JupyterHub: only pushed to Docker Hub as that still works, but I think we should switch to pushing releases to both quay.io and Docker Hub, and dev images to quay.io.